src/main/java/ai/elimu/web/content/allophone/AllophoneListController.java

```diff
@@ -90,7 +90,7 @@
         {"ɔ","O", "0"},
         {"u","u", "142"},
         {"m","m", "124"},
-        {"ɓ","b_<", "68"},
+        {"ɓ","b__", "68"},
```

Comment:
Instead of changing the X-SAMPA value to an invalid value, I suggest commenting out the line.